lib/tokenizers.js

```diff
@@ -37,20 +41,33 @@ bedrock.events.on('bedrock-mongodb.ready', async () => {
   }]);
 });
 
-export async function get({id} = {}) {
+export async function get({id, explain = false} = {}) {
   if(!(id && typeof id === 'string')) {
     throw new TypeError('"id" must be a string.');
   }
 
   // return cached tokenizer if available
-  if(CACHED_TOKENIZER && CACHED_TOKENIZER.id === id) {
+  if(!explain && CACHED_TOKENIZER && CACHED_TOKENIZER.id === id) {
     return CACHED_TOKENIZER;
   }
 
+  // the query of 'tokenizer.id' is properly indexed. No additional queries
+  // should be added here without ensuring that additional queries are properly
+  // indexed first.
   const query = {'tokenizer.id': id};
   const projection = {_id: 0};
   const collection = database.collections['tokenizer-tokenizer'];
-  const record = await collection.findOne(query, projection);
+
+  if(explain) {
+    // this is used to explain the results of the query in order to ensure that
+    // the query is using the proper indexes. 'find().limit(1)' is used here
```

Comment:
This comment presumes that there's only one use case for passing in `explain`. I don't think we want or need the first part of this comment. The second part about how `find().limit(1)` is used is helpful because it's not common knowledge and may never become common.